src/qmapshack/gis/rte/router/CRouterRoutino.cpp

```diff
@@ -300,13 +327,28 @@ void CRouterRoutino::calcRoute(const IGisItem::key_t& key)
         {
             throw QString();
         }
-
-        Routino_Database * data = (Routino_Database*)comboDatabase->currentData(Qt::UserRole).toULongLong();
+        int res = 0;
+        QVariantMap map = comboDatabase->currentData(Qt::UserRole).toMap();
+        Routino_Database * data = (Routino_Database*)(map["db"].toULongLong());
         if(nullptr == data)
         {
             throw QString();
         }
 
+        // load xml profiles if it is new before setting the profile!
+        QString prefixpath = map["prefixpath"].toString();
+        if( currentPrefix != prefixpath)
+        {
+            currentPrefix = prefixpath;
+            printf("cR CGIR PrefixPath %s",prefixpath.toStdString().data());
```

Comment:
What's that good for? For debug output use `qDebug()` as the rest of the code does.

Reply:
Redundant - thought I had changed all  into qdebug.